### PR TITLE
Some fixes for log Actions and Sending emails

### PR DESCRIPTION
**Pull request by WilcoLouwerse**
Also added a log email template

(on my phone the Conduction logo is actually visible 😢)
image

### CODE REVIEW

**Review by bbrands02 — APPROVED** [review]

File: api/src/Logger/SessionDataProcessor.php [hunk 1]
```diff
@@ -81,6 +81,7 @@ public function dispatchLogCreateAction(array $record): array
                 $this->entityManager->getConnection()->getSchemaManager()->listDatabases()
             ) === true
             && $this->entityManager->getConnection()->getSchemaManager()->tablesExist('action') === true
+            && in_array($record['level_name'], ['DEBUG', 'INFO']) === false
```
Comment (rjzondervan) [inline api/src/Logger/SessionDataProcessor.php:84]: I think we should only include ERROR and CRITICAL here, Warning and Notice are good to log, but shouldn't result in e-mails I think.

File: api/templates/emails/gateway/new-log-e-mail.html.twig [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+                            <tr></tr>
+                        </table>
+                    </td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+</table>
+</body>
+</html>
```
Comment (rjzondervan) [inline api/templates/emails/gateway/new-log-e-mail.html.twig:503]: Seems alright, but shouldn't this be in configuration like the way Huwelijksplanner does this?

**Review by rjzondervan — CHANGES_REQUESTED** [review]
Two points to attend, one about hardcode vs configurable, one about log levels to send emails for.

File: api/src/Logger/SessionDataProcessor.php (hunk 1 above)
Comment (WilcoLouwerse) [reply api/src/Logger/SessionDataProcessor.php:84]: Yeah you might be right about that, i mostly used NOTICE for testing in the beginning, but I do not expect that to be a used for email sending either

Comment (WilcoLouwerse) [reply api/src/Logger/SessionDataProcessor.php:84]: We also have ALERT and EMERGENCY, so i will just add NOTICE and WARNING to this list for now

File: api/templates/emails/gateway/new-log-e-mail.html.twig (hunk 2 above)
Comment (WilcoLouwerse) [reply api/templates/emails/gateway/new-log-e-mail.html.twig:503]: ah, adding it as Action to a Bundle.
To which bundle should I add this Action?

Comment (rjzondervan) [reply api/templates/emails/gateway/new-log-e-mail.html.twig:503]: For the moment you could think about one of the VrijBRP bundles (don't have the names on the top of my head right now)

**Review by rjzondervan — APPROVED** [review]

**Pull request merged**